### ISSUE TITLE
Disable web3 timeout when sending a transaction

### ISSUE THREAD

**Issue by pandadefi** [ApeWorX/ape#807]
### Overview

I am deploying multiple contracts with a script and the script stopped with the following error:
```
web3.exceptions.TimeExhausted: Transaction HexBytes('0xd3f058a67f1569e2d0f41ed6f017c92c8e95441448bd3507bd44058940c113ae') is not in the chain after 120 seconds
```

I wouldn't mind waiting a little more and have my script completed.

**Comment by fubuloubu** [comment]
A timeout override would be nicer, yeah

**Comment by unparalleled-js** [comment]
If you are using Hardhat, you can its `timeout`:

```yaml
hardhat:
  request_timeout: 20  # Defaults to 30
  fork_request_timeout: 600  # Defaults to 300
```

**Comment by pandadefi** [comment]
It's to deploy on mainnet. 
Having a script that fails half way into the deployment because the gas price went up a little and the transaction takes more than 120 seconds to get mined shouldn't happen.
Best of the best would be to suggest a rebroadcast with increased fee

**Comment by NotPeopling2day** [comment]
Related ticket https://github.com/ApeWorX/ape/pull/930

**Comment by antazoey** [comment]
fixed by #930 you can also use config
